Barbara

Barbsmontague opened the pull request:
Added in Barbaramontague-advice page but not happy that the style file is linked or not?? Any suggestions??

redrambles commented:
Hey @Barbsmontague !

This is great!!! The only thing we need you to do before we can merge in this pull request is to delete the `css/barbaramontague-style.css` file. You have placed some CSS in the main stylesheet (css/styles.css) and are linking to that from your advice page, so we don't need this extra stylesheet.

Once you have deleted css/barbaramontague-style.css - please add (`git add -A`), commit (`git commit -m "your message here"`) and then push up your branch once more (`git push origin Barbara`), which will automatically update this pull request.

Then we should be all set to merge in your code!

Let us know if you have any questions!

Cheers,
Ann

Barbsmontague commented:
Thank you Ann,

I have deleted file and added, commit and pushed files back to you. I do
have a query though as it seems there is an error in linking my css file to
my html. as shown in the attached picture the code is grey instead of
orange in text writer ATM and when I tried to validate the css file it says
".barbaramontague-advice is a CSS hack"  What does this mean and how do I
fix it? Thanks for your help
Cheers Barbara

On Sat, Sep 8, 2018 at 1:32 AM Ann Cascarano <notifications@github.com>
wrote:

> Hey @Barbsmontague <https://github.com/Barbsmontague> !
>
> This is great!!! The only thing we need you to do before we can merge in
> this pull request is to delete the css/barbaramontague-style.css file.
> You have placed some CSS in the main stylesheet (css/styles.css) and are
> linking to that from your advice page, so we don't need this extra
> stylesheet.
>
> Once you have deleted css/barbaramontague-style.css - please add (git add
> -A), commit (git commit -m "your message here") and then push up your
> branch once more (git push origin Barbara), which will automatically
> update this pull request.
>
> Then we should be all set to merge in your code!
>
> Let us know if you have any questions!
>
> Cheers,
> Ann
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/skillcrush-104/pull/2115#issuecomment-419477397>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ApB55qrH1U2qB7Go_VeJUa7qj5Jpwuhyks5uYpF4gaJpZM4WcPwc>
> .
>
